src/Symfony/Component/Config/Tests/FileLocatorTest.php

```diff
@@ -86,7 +87,7 @@ public function testLocate()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException FileLocatorFileNotFoundException
```

Comment:
phpunit doesn't handle use statements, should be `Symfony\Component\Config\Exception\FileLocatorFileNotFoundException` (same below)